test/unit/sql/create-table.test.js

```diff
@@ -30,6 +30,11 @@ describe(Support.getTestDialectTeaser('SQL'), () => {
           mariadb: 'CREATE TABLE IF NOT EXISTS `foo`.`users` (`id` INTEGER NOT NULL auto_increment , `mood` ENUM(\'happy\', \'sad\'), PRIMARY KEY (`id`)) ENGINE=InnoDB;',
           mysql: 'CREATE TABLE IF NOT EXISTS `foo.users` (`id` INTEGER NOT NULL auto_increment , `mood` ENUM(\'happy\', \'sad\'), PRIMARY KEY (`id`)) ENGINE=InnoDB;',
           mssql: 'IF OBJECT_ID(\'[foo].[users]\', \'U\') IS NULL CREATE TABLE [foo].[users] ([id] INTEGER NOT NULL IDENTITY(1,1) , [mood] VARCHAR(255) CHECK ([mood] IN(N\'happy\', N\'sad\')), PRIMARY KEY ([id]));',
+          ibmi: `BEGIN
```

Comment:
Any reason the changes in these files are on separate lines?

Reply:
The query generator itself generates these queries as multi-line queries. I did this for readability, as I find:

```
BEGIN
  DECLARE CONTINUE HANDLER FOR SQLSTATE VALUE '42891'
  BEGIN END;
  ALTER TABLE "User" ADD CONSTRAINT "a_b_uniq" UNIQUE ("fieldB", "fieldA" DESC);
END
```
a lot easier to conceptualize than
```
BEGIN DECLARE CONTINUE HANDLER FOR SQLSTATE VALUE '42891' BEGIN END; ALTER TABLE "User" ADD CONSTRAINT "a_b_uniq" UNIQUE ("fieldB", "fieldA" DESC); END
```